src/components/layout/nav-group.tsx

```
@@ -0,0 +1,179 @@
+import {
+    DropdownMenu,
+    DropdownMenuContent,
+    DropdownMenuItem,
+    DropdownMenuLabel,
+    DropdownMenuSeparator,
+    DropdownMenuTrigger,
+} from '../ui/dropdown-menu'
+import { NavCollapsible, NavItem, NavLink, type NavGroup } from './types'
+import {Link} from "react-router-dom";
+
+export function NavGroup({ title, items }: NavGroup) {
```

Biome (1.9.4)

[error] Shouldn't redeclare 'NavGroup'. Consider to delete it or rename it.

'NavGroup' is defined here:


(lint/suspicious/noRedeclare)